tests/integ/test_index.py

```diff
@@ -8,128 +8,131 @@
 from pinecone import ApiTypeError, ApiException
 
 responses_req = Responses()
-responses = Responses('requests.packages.urllib3')
+responses = Responses("requests.packages.urllib3")
 
+@pytest.mark.skip()
```

Comment:
There are a lot of black changes in this file, but all the tests are currently marked `skip` because they can't pass until we overhaul configuration.

The urllib3_mock we are depending on throws a lot of deprecation warnings and appears to be abandoned so we should figure out some other tool there if we need to do request mocking in the future. But in reality I don't think we need to mock requests at that level if we refactor our tests into either pure unit tests or real integration tests against a live environment.